designsafe/static/scripts/data-depot/components/projects/pipeline-publish/pipeline-publish.component.js

```diff
@@ -59,12 +58,12 @@ class PipelinePublishCtrl {
                 let entity = this.project.getRelatedByUuid(uuid);
```

Comment:
There is an inconsistant bug when clicking the "Request DOI and Publish" button. The button hangs while loading the `pipeline-publish` component and it seems to have an issue looking up an **entity uuid** from the list of `selectedListings` Error: Cannot read property 'name' of undefined. I can't find what is causing the issue yet, but it only happens with experimental, simulation, field research, or hybrid sim type projects. I have confirmed that this is not happening in master. I'm planning to dig into this more.